src/app/features/tasks/task-list/task-list.component.ts

```diff
@@ -122,23 +122,8 @@ export class TaskListComponent implements OnDestroy, AfterViewInit {
   }
 
   enterPredicate(drag: CdkDrag, drop: CdkDropList): boolean {
-    // TODO this gets called very often for nested lists. Maybe there are possibilities to optimize
-    const task = drag.data;
-    // const targetModelId = drag.dropContainer.data.listModelId;
-    const targetModelId = drop.data.listModelId;
-    const isSubtask = !!task.parentId;
-    // console.log(drag.data.id, { isSubtask, targetModelId, drag, drop });
-    // return true;
-    if (isSubtask) {
-      if (!PARENT_ALLOWED_LISTS.includes(targetModelId)) {
```

Comment:
I don't think we can remove this, since this handles the logic that prevents some illegal drag and drop operations.